hubblestack/daemon.py

```diff
@@ -820,6 +822,28 @@ def clear_selective_context():
     # clear the package list so that pkg module can fetch it as fresh in next cycle
     __context__.pop('pkg.list_pkgs', None)
 
+def setup_fips_mode():
+    """
+    Setup FIPS mode
+    """
+    global __opts__
+    fips_mode_enable = __opts__.get('fips_mode', False)
+
+    if not fips_mode_enable:
+        print('FIPS mode not configured')
+        return
+
+    if hubblestack.utils.platform.is_windows():
```

Comment:
We should check what was set in hubble config for windows too.

Reply:
It is checked on top of this function for all. Its not specific to windows/linux